source/mapping-rules/fields/data-type-conversion-guide.txt

```diff
@@ -0,0 +1,229 @@
+.. _rm-data-type-conversion-guide:
+
+==============================
+Data Type Conversion Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page outlines examples that show the result of converting the 
+source value and data type to a specific BSON data type using customized 
+fields.
+
+Behavior
+~~~~~~~~
+
+- Sync job errors result in ``BsonNull`` values for the impacted fields. 
+  Your field's :guilabel:`Null handlings` strategy does not affect this 
+  behavior.
+
+- Your sync job continues with errors unless it reaches the user-defined 
+  threshold.
```

Comment:
[suggestion] Should we link to the place where we describe how to set this threshold?

Reply:
This is documented, but its part of the create sync job task. For simplicity I think this should be left as is to avoid confusing the users.